lib/src/commands/OptionsProcessor.test.ts

```diff
@@ -135,4 +135,14 @@ describe('navigation options', () => {
     expect(options.topBar.title.component.passProps).toBeUndefined();
     expect(options.topBar.background.component.passProps).toBeUndefined();
   });
+
+  it('calls store when button has passProps and id', () => {
```

Comment:
This feature is relevant not only to buttons, but to any component, even screens. Lets rename `button` to `component`